src/libraries/System.Runtime/tests/System/TimeZoneInfoTests.cs

```diff
@@ -2424,6 +2424,17 @@ public static void TimeZoneDisplayNames_Unix(TimeZoneInfo timeZone)
             }
         }
 
+        [Fact]
+        [PlatformSpecific(~TestPlatforms.Windows)]
+        public static void UtcAliases_MapToUtc()
+        {
+            foreach (string utcAlias in s_UtcAliases)
+            {
+                TimeZoneInfo actualUtc = TimeZoneInfo.FindSystemTimeZoneById(utcAlias);
```

Comment:
please check the BaseUtcOffset too.

Reply:
`TimeZoneInfo.HasSameRules` is a convenient method which compares BaseUtcOffset, daylight saving time, and adjustment rules. The only thing it doesn't compare is Id. In fact, `TimeZoneInfo.Equals` is implemented as
``` csharp
public bool Equals([NotNullWhen(true)] TimeZoneInfo? other) =>
    other != null &&
    string.Equals(_id, other._id, StringComparison.OrdinalIgnoreCase) &&
    HasSameRules(other);
```